src/configure.ac

```diff
@@ -1091,7 +1036,7 @@ if test "`echo *_plug.c`" != "*_plug.c"; then
    AC_SUBST([OPENCL_PLUGFORMATS_OBJS],[`echo opencl*_plug.c | sed 's/\.c/.o/g'`])
    if test "x$plug_deps" = xyes -a "x$PERL" != x ; then
       AC_MSG_NOTICE([creating Makefile dependencies])
-      AC_SUBST([PLUGFORMATS_DEPS],[`$PERL ./plugin_deps.pl *_plug.c`])
+      AC_SUBST([PLUGFORMATS_DEPS],[`$PERL ./plugin_deps.pl 2>/dev/null *_plug.c`])
```

Comment:
Is the `2>/dev/null` related to other changes in this PR or just happens to have been added at the same time - and why?

Reply:
My dodgy `plugindeps.pl` can't detect includes that are ifdef'ed out, so it would give a host of warnings for files not copied from mbetTLS, but still "referenced".

Reply:
I think this is worth adding a comment on.

Reply:
Do you mean right there in `configure.ac` or what?

Reply:
Yes, add a comment near that line in `configure.ac` or inside the Perl script.